routes/cube_routes.js

```diff
@@ -4174,4 +4285,22 @@ router.get(
   }),
 );
 
+router.get(
+  '/api/date_updated/:id',
+  util.wrapAsyncApi(async (req, res) => {
+    const { id } = req.params;
+    const result = await Cube.findOne(buildIdQuery(id), { date_updated: 1, _id: 0 }).lean();
```

Comment:
```suggestion
    const result = await Cube.findOne(buildIdQuery(id), 'date_updated').lean();
```
I'd prefer to keep projection consistent across code base

Reply:
Read more here: https://mongoosejs.com/docs/api.html#query_Query-select